CASMPET-5786 1.3 : DOCS: Update postgres kubectl exec/logs calls to include postgres container (#2047)
Cray-HPE/docs-csm: operations/kubernetes/View_Postgres_Information_for_System_Databases.md

    run: /etc/service/pgqd: (pid 25) 487273s
    ```

> 2. Log in as the `postgres` user.

Reviewer: ```suggestion
1. Log in as the `postgres` user.
```

> 1. (`cray-smd-postgres#`) Log in as the `postgres` user.

Reviewer: ```suggestion
1. (`postgres-container#`) Log in as the `postgres` user.
```

    ```bash
    root@cray-smd-postgres-0:/home/psql -U postgres
    ```

    Example output:

    ```text
    psql (12.2 (Ubuntu 12.2-1.pgdg18.04+1), server 11.7 (Ubuntu 11.7-1.pgdg18.04+1))
    Type "help" for help.

    postgres=#
    ```

> 3. List the existing databases.

Reviewer: ```suggestion
1. List the existing databases.
```

    Example output:

    ```text
    psql (12.2 (Ubuntu 12.2-1.pgdg18.04+1), server 11.7 (Ubuntu 11.7-1.pgdg18.04+1))
    You are now connected to database "hmsds" as user "postgres".
    hmsds=#
    ```

> 5. List the data types that are in the database being viewed.

Reviewer: ```suggestion
1. List the data types that are in the database being viewed.
```
